packages/wrangler/src/metrics/metrics-dispatcher.ts

```diff
@@ -67,6 +67,7 @@ export async function getMetricsDispatcher(options: MetricsConfigOptions) {
 			properties: {
 				category: "Workers",
 				wranglerVersion,
+				os: process.platform + ":" + process.arch,
```

Comment:
not related to this PR, but could you add this to our useragent as well?

Reply:
Let's do that in a follow up PR

Reply:
Here is a link to the standard format of UserAgent that could be referenced: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent